vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/Spreadsheet.java

```diff
@@ -2644,12 +2645,12 @@ private void fireRowHeaderDoubleClick(int rowIndex) {
      *
      * @param listener
      *            The listener to add
+     * @return a {@link Registration} for removing the event listener
      **/
-    public void addRowHeaderDoubleClickListener(
+    public Registration addRowHeaderDoubleClickListener(
```

Comment:
nit: we could move this method closer to the other listeners, maybe closer to the `RowHeaderDoubleClickEvent` declaration.

Reply:
Done